packages/cli/src/commands/deploy-v2.ts

```diff
@@ -63,17 +63,15 @@ export async function deployHandler(args: Parameters<(typeof commandModule)["han
     // Get the basename of the file
     .map((path) => basename(path, ".sol"));
 
-  // Load and resolve the config
-  const worldConfig = await loadWorldConfig(configPath, existingContracts);
-  const storeConfig = await loadStoreConfig(configPath);
-  const mudConfig = { ...worldConfig, ...storeConfig };
+  // Load the config
+  const mudConfig = (await loadConfig(configPath)) as MUDCoreConfig;
```

Comment:
I noticed `MUDCoreConfig` here includes all types from store's and world's `typeExtensions` - how does that work? it it because they're part of the same monorepo?